examples/group_provisioning_demo.c

```
@@ -0,0 +1,275 @@
+        memcpy(join.group_key, group_key, SYMMETRIC_KEY_LENGTH);
+
+        packet_t pkt;
+        if (packet_create(&pkt, MEMBER_A, PACKET_TYPE_GROUP_JOIN,
+                          (uint8_t*)&join, sizeof(join)) != 0) {
+            fprintf(stderr, "[FAIL] Create GROUP_JOIN packet for A\n");
+            return 1;
+        }
+
+        /* Encrypt using group key here just for demo integrity check;
+         * in real code this would use the pairwise session key. */
+        if (packet_encrypt(&pkt, group_key) != 0) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)